ical/parsing/component.py

```diff
@@ -127,3 +127,10 @@ def parse_content(content: str) -> list[ParsedComponent]:
 def encode_content(components: list[ParsedComponent]) -> str:
     """Encode a set of parsed properties into content."""
     return "\n".join([component.ics() for component in components])
+
+
+def unfolded_lines(content: str) -> Generator[str, None, None]:
+    """Read content and unfold lines."""
+    content = FOLD_RE.sub("", content)
+    for line in LINES_RE.split(content):
+        yield line
```

Comment:
you can do `yield from LINES_RE.split(content)`

Reply:
Nice, thanks, updated.